### DISCUSSION TITLE
would a community maintained wiki be a useful addition to discussions?

### DISCUSSION THREAD

**Discussion by tg0h** [kovidgoyal/kitty#5378] [General]
I wonder if a community maintained wiki similar to that in the fzf repo https://github.com/junegunn/fzf/wiki would be a good place to store useful tips and recipes.

Some useful information in PRs or discussions which would be easier to find if they were in a wiki
- using nvim as a scrollback pager - recipe mentioned in a github issue i believe
- how to set up a custom tab bar - there is a github discussion on this
- running a script from a key and not having it exit immediately - from a github Issue I recently found, which mentions a hint to use a semicolon when launching a shell `launch zsh -c "<command>; zsh"` 

The wiki can also serve as a cookbook for users to show off useful recipes they use.

**Comment by kovidgoyal** [comment]
Sounds fine to me, as long as I dont have to maintain it :)

**Reply by tg0h** [reply]
If that is the case, let's open up access to the community and we'll see whether it takes off?

![image](https://user-images.githubusercontent.com/7954426/184529540-f1976140-c674-4d02-add4-d610e9afce35.png)

**Comment by kovidgoyal** [comment]
before opening up access to it, it would be nice if someone posted here saying they are willing to contribute something to the wiki to get the ball rolling.

**Comment by page-down** [comment]
I would avoid using the GitHub Wiki, it's harder to use than a git repository, and if I remember correctly, the content in the wiki can't be found by search engines.
GitHub ignores these issues and hasn't improved anything over the years.
Using a platform's proprietary features is not beneficial in the long run.
